beacon-chain/sync/pending_attestations_queue_test.go

```diff
@@ -703,41 +703,3 @@ func Test_attsAreEqual_Committee(t *testing.T) {
 		assert.Equal(t, false, attsAreEqual(att1, att2))
 	})
 }
-
-func Test_SeenCommitteeIndicesSlot(t *testing.T) {
```

Comment:
This test doesn't add anything new. Having `TestService_setSeenUnaggregatedAtt` is sufficient